### ISSUE TITLE
The <colModal> parameters can ignore so much the better

### ISSUE THREAD

**Issue by zyz** [meetselva/fixed-table-rows-cols#31]
Too many number of columns, This parameter must be very inconvenient.
 for example: 30 to 50 columns

**Comment by meetselva** [comment]
Yes, it would be troublesome to configure each and every column.. Where do you set the column width? Letting the browser auto-set the width can be a problem.

**Comment by jxramos** [comment]
I think it's becoming clearer to me that this solution really does lean on the choice of colModal values. I'm providing it with a best guess passed down into jinja template generation but that's a funky hack. It would be great to generalize this solution somehow to autofit the body and header columns to be the same width much like how the body rows are used to update the fixed column's row heights.
